aws/resource_aws_launch_configuration_test.go

```diff
@@ -735,6 +764,22 @@ resource "aws_launch_configuration" "test" {
 `, rInt, rInt, rInt)
 }
 
+func testAccAWSLaunchConfigurationConfigMetadataOptions(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_launch_configuration" "test" {
+  name = %[1]q
+  image_id = "${data.aws_ami.ubuntu.id}"
+  instance_type = "t3.nano"
+
+  metadata_options {
+    http_endpoint               = "enabled"
+    http_tokens                 = "required"
+    http_put_response_hop_limit = 2
+  }
+}
+`, rName)
```

Comment:
This test configuration was missing the previously available `testAccAWSLaunchConfigurationConfig_ami()` for the `data.aws_ami.ubuntu.id` reference. Since this resource testing was converted to `testAccLatestAmazonLinuxHvmEbsAmiConfig()` since submission, we can correct this here via:

```suggestion
	return composeConfig(
		testAccLatestAmazonLinuxHvmEbsAmiConfig(),
		fmt.Sprintf(`
resource "aws_launch_configuration" "test" {
  image_id      = data.aws_ami.amzn-ami-minimal-hvm-ebs.id
  instance_type = "t3.nano"
  name          = %[1]q

  metadata_options {
    http_endpoint               = "enabled"
    http_tokens                 = "required"
    http_put_response_hop_limit = 2
  }
}
`, rName))
```